game/scripts/vscripts/Sections/Chapter4/SectionOpening.ts

```diff
@@ -157,16 +159,20 @@ function onStart(complete: () => void) {
 
             tg.audioDialog(LocalizationKey.Script_4_Opening_8, LocalizationKey.Script_4_Opening_8, ctx => ctx[CustomNpcKeys.SlacksMudGolem]),
             tg.audioDialog(LocalizationKey.Script_4_Opening_9, LocalizationKey.Script_4_Opening_9, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
-            tg.audioDialog(LocalizationKey.Script_4_Opening_10, LocalizationKey.Script_4_Opening_10, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
+            tg.fork([
+                tg.audioDialog(LocalizationKey.Script_4_Opening_10, LocalizationKey.Script_4_Opening_10, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
+                tg.immediate(_ => {
```

Comment:
Could just put that before the audio dialog and get rid of the fork

Reply:
I guess you can, it's effectively the same thing.